conan/solo5/0.4.1/conanfile.py

```diff
@@ -11,7 +11,7 @@ class Solo5Conan(ConanFile):
 
     def source(self):
         repo = tools.Git(folder = self.name)
-        repo.clone(self.url + ".git",branch="v{}".format(self.version))
+        repo.clone(self.url + ".git",branch="master")
```

Comment:
This file should live in the conan repo please update there. Do we depend on an unstable trunk ?  (replace with scm ?)